coremltools/converters/mil/mil/ops/defs/pool.py

```diff
@@ -77,8 +77,8 @@ def type_inference(self):
 @register_op(doc_str="")
 class avg_pool(Pooling):
     """
-    Perform average pooling. Currently defined only for spatial 1-D and
-    2-D cases. (Can be extended to the 3-D case easily.)
+    Perform average pooling. Supports spatial 1-D, 2-D, and 3-D
```

Comment:
Supports 1-D, 2-D and 3-D pool (1, 2 or 3 spatial dimensions)

Reply:
OK